go.mod

```
@@ -1,4 +1,4 @@
-module github.com/deepmap/oapi-codegen/v2
+module github.com/oapi-codegen/oapi-codegen
```

:stop_sign: we're still gonna need the `/v2` in here

```suggestion
module github.com/oapi-codegen/oapi-codegen/v2
```